internal/ingress/defaults/main.go

```diff
@@ -100,7 +100,7 @@ type Backend struct {
 
 	// Name server/s used to resolve names of upstream servers into IP addresses.
 	// The file /etc/resolv.conf is used as DNS resolution configuration.
-	Resolver []net.IP
+	Resolver []net.IP `json:"Resolver"`
```

Comment:
I'm not sure we use this field on some operation, so instead of setting it on the json output can we try ignoring it? (`json:"-"` should do the job)